### PR TITLE
now enable the user to drag using the whole screen

### PR DESCRIPTION
**Pull request by wouter-willems**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "mm-image-cropper",
-  "version": "0.0.1",
+  "version": "1.1.1",
```
Comment (wouter-willems) [inline package.json]: previous package was outdated, so this change is legit

File: src/index.js [hunk 2]
```diff
@@ -827,35 +826,17 @@ export default angular.module("mm.image-cropper", []).directive('mmImageCrop', [
...
-			};
-
-			element.bind('mousemove', mousemove);
 			element.bind('mousedown', mousedown);
-			element.bind('mouseup', mouseUp);
-
-			element.bind('touchstart', touchDown);
-			element.bind('touchmove', touchMove);
-			element.bind('touchend', touchUp);
+			document.addEventListener("mousemove", function(e){
```
Comment (AlexKolpa) [inline src/index.js]: Not sure how this works for event listeners, but shouldn't these be removed manually as well?

**Comment by AlexKolpa** [comment]
If you can answer my question :+1:

**Comment by wouter-willems** [comment]
You were right, I changed it so no new listeners are added everytime

**Comment by TiddoLangerak** [comment]
I don't know what you changed since you rebased, but the listeners are still not detached, as @AlexKolpa requested. You should still do that.

**Pull request merged**